TelCo.ColorCoder/NumberToColor.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Diagnostics;
+using System.Drawing;
+
+namespace TelCo.ColorCoder
+{
+    class NumberToColor{
+
+         /// <summary>
+        /// Given the two colors the function returns the pair number corresponding to them
+        /// </summary>
+        /// <param name="pair">Color pair with major and minor color</param>
+        /// <returns></returns>
+        public static int GetPairNumberFromColor(ColorPair pair)
```

Comment:
Another minor comment, again on Naming. A good way to name your classes and functions is by seeing how it would be written from the place its called.
Here it would be NumberToColor.GetPairNumberFromColor 
So here "NumberToColor" is repeated at class level and function level, which is not idealistic. 
This is to keep in mind for naming convention, do not have to fix this.